### PR TITLE
fix e2e

### PR DESCRIPTION
**Pull request by jpinsonneau**
## Description

Fix e2e:
- `gRPC` case:
  - renamed to `IfDirections` and `Interfaces` and check for arrays
- `IPFIX` case:
  - check for `FlowDirection` and `Interface`
  - updated flp dependency since it's involved in this e2e test

I would recommend to move away these tests to a new dedicated repo to avoid going back and forth updating dependencies.
These could also include console plugin cypress tests, operator deployment and CLI in future.

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [ ] Will this change affect NetObserv / Network Observability operator? If not, you can ignore the rest of this checklist.
* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [x] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: e2e/basic/common.go [hunk 1]
```diff
@@ -66,8 +65,6 @@ func (bt *FlowCaptureTester) DoTest(t *testing.T) {
 			assert.Less(t, flow["Packets"], float64(10))
 			assert.Less(t, time.Since(asTime(flow["TimeFlowEndMs"])), 15*time.Second)
 			assert.Less(t, time.Since(asTime(flow["TimeFlowStartMs"])), 15*time.Second)
-
-			assert.NotEmpty(t, flow["Interface"])
```
Comment (jpinsonneau) [inline e2e/basic/common.go:70]: I don't know why yet but I'm getting <nil> in `IfDirections` and `Interfaces` here :thinking: 

I'm investigating

**Comment by msherif1234** [comment]
/lgtm
not sure why u updated FLP in this PR but have no prob with that

**Comment by jpinsonneau** [comment]
> /lgtm assuming e2e will pass, also I am not sure why u updated FLP in this PR but have no prob with that

@msherif1234 I have issues with IPFIX export, I wonder if it's broken :thinking: 
I get nil even if I check for old fields

**Pull request merged**